test/integration-tests/UniswapMixed.test.ts

```diff
@@ -35,6 +47,13 @@ describe('Uniswap V2 and V3 Tests:', () => {
   let wethContract: Contract
   let usdcContract: Contract
   let planner: RoutePlanner
+  let v4Planner: V4Planner
+  let v4PoolManager: PoolManager
+  let v4PositionManager: PositionManager
+
+  // current market ETH price at block
+  const USD_ETH_PRICE = 3820
+  const ONE_PERCENT = 38
```

Comment:
nit: this is 1% OF the pinned usd_eth_price so I feel like you can just calculate it in line using `USD_ETH_PRICE` I just got confused reading this bc its not "one percent"literally